### DISCUSSION TITLE
Modifications needed to popup window

### DISCUSSION THREAD

**Discussion by Dan-Essin** [sienori/Tab-Session-Manager#1047] [General]
Hello,
Nice extension but awkward to use. The window is not resizable and is too narrow to see the really important buttons - edit and delete.
I suggest that you eliminate the second and third icons  on the left  and move the edit and delete icons to the left side. Icons 2 and 3 are cute but waste valuable space.
For me edit and delete will be the ones to use regularly. Having to use the scrollbar on every line if I want to close a bunch of tabs is a real pain. Also, by the time you have scrolled to the right, most of the tab label becomes invisible so you can't really tell which tab you wanted to delete.

I would also suggest to remove the confirm tab close question. If I close one my accident they are all in the history. This extra click is serves no useful purpose, imho. It's just annoying.

Bottom line - This extension is far better than 2 others I tried that fail to list the tabs, but without the changes that I have suggested I will find it annoying and may end up not using it. It has real potential and I hope that you agree with my critique and make some modifications.

Thank you
![IMG_0591](https://user-images.githubusercontent.com/1776053/186019661-6eaa1bad-e4e3-490a-8e4e-008a9b293e01.jpg)
![IMG_0593](https://user-images.githubusercontent.com/1776053/186021016-c434f5c1-b3e2-4406-b323-2f6fc9590620.jpg)

**Comment by sienori** [comment]
Your screenshot is not Tab Session Manager...
